packages/venia-ui/lib/components/Routes/routes.js

```diff
@@ -1,10 +1,14 @@
 import React, { Suspense } from 'react';
 import { Route, Switch, useLocation } from 'react-router-dom';
 
+import { useScrollTopOnChange } from '@magento/peregrine/lib/hooks/useScrollTopOnChange';
 import { fullPageLoadingIndicator } from '../LoadingIndicator';
 import HomePage from '../HomePage';
 import MagentoRoute from '../MagentoRoute';
-import { useScrollTopOnChange } from '@magento/peregrine/lib/hooks/useScrollTopOnChange';
+
+// Import used for makeRoutesTarget
+// eslint-disable-next-line no-unused-vars
+import AuthRoute from './authRoute';
```

Comment:
It would be preferable to add the import to `makeRoutesTarget`, and we can remove these lines.

```js
// function addRoutes
const AuthRouteComponent = routeList.addImport(
    'import AuthRoute from "./authRoute"'
);

...

const Component = route.authed ? AuthRouteComponent : 'Route';
```

Reply:
@tjwiebell Done. Thanks for the code!

Reply:
Oh no, my pseudo code didn't make this clear, sorry 🙂 the `AuthRouteComponent` should be outside of the for loop; I don't think there's any reason to have a unique import per route.

Reply:
Yeah, I should have figured that out myself. 😅 
I just moved it.